wazuh/wazuh-kibana-app
# Wazuh main menu is displayed when the navigation menu is locked

### Description
Change the z-index value of the Wazuh menu class so that it is displayed correctly when the navigation menu is locked.
 
### Issues Resolved
- #5264 

### Evidence

![image](https://user-images.githubusercontent.com/63758389/223787791-3c0bcb54-65e9-465c-b7ab-dd029c2a6f77.png)

![image](https://user-images.githubusercontent.com/63758389/223787844-8706f9fa-1b8b-4908-9f1f-be697663b5d1.png)

![image](https://user-images.githubusercontent.com/63758389/223787880-a7384efa-7764-40dd-b139-5511e50c871a.png)


### Test

1. Open the navigation menu and then the wazuh menu.
Should see the wazuh menu at the back of the navigation menu.

2. Open the navigation menu, lock it and then the wazuh menu.
Should see the wazuh menu in front of the navigation menu.

### Check List
